src/cpp/core/r_util/RTokenizer.cpp

```diff
@@ -463,9 +463,47 @@ RToken RTokenizer::consumeToken(RToken::TokenType tokenType,
                  column);
 }
 
+class ConversionCache
```

Comment:
@jjallaire, this little class exists to cache the results of converting between wide and UTF8 (as we may perform the conversion for the same token many times, since tokenization is done on wide strings). Do you think this is sensible?